### PR TITLE
check node installed before run ssh

### PR DESCRIPTION
**Pull request by yungez**
1. check node installed before run ssh
2. for cloudshell usage promopt, add 'not show this again' ability
3. cloudshell connection keepalive

### CODE REVIEW

File: src/cloudShellRunner.ts [hunk 1]
```diff
@@ -52,50 +52,67 @@ export class CloudShellRunner extends BaseRunner {
 
     protected startCloudShell(playbook: string): void {
         const msgOption: vscode.MessageOptions = { modal: false };
-        const msgItem: vscode.MessageItem = { title: 'Confirm' };
+        const msgItem: vscode.MessageItem = { title: 'Confirm & Not show this again' };
```
Comment (zikalino) [inline src/cloudShellRunner.ts]: just little thing, i think Not should be lowercase --> "Confirm and don't show this again"
or actually maybe better.

File: src/cloudShellRunner.ts [hunk 2]
```diff
@@ -52,50 +52,67 @@ export class CloudShellRunner extends BaseRunner {
...
+                                    terminal.sendText('export ' + Constants.UserAgentName + '=' + utilities.getUserAgent());
                                 }
-                            }, 500);
+                                terminal.sendText('ansible-playbook ' + path.basename(playbook));
+                                terminal.show();
+
+                                count = 0;
+                            }
                         } else {
-                            this._outputChannel.appendLine('\nConnecting to cloud shell failed, please retry.');
```
Comment (zikalino) [inline src/cloudShellRunner.ts]: Let's just be consistend with cloud shell naming. Sometimes it's "cloud shell", sometimes "Cloudshell".
Maybe we should be consistent with this doc: https://docs.microsoft.com/en-us/azure/cloud-shell/overview
and just have "Cloud Shell" everywhere

File: src/cloudShellRunner.ts (hunk 1 above)
Comment (yungez) [reply src/cloudShellRunner.ts]: i think & is more obvious..

File: src/cloudShellRunner.ts (hunk 2 above)
Comment (yungez) [reply src/cloudShellRunner.ts]: nice. fixed

File: src/cloudShellRunner.ts (hunk 1 above)
Comment (zikalino) [reply src/cloudShellRunner.ts]: ok so just add lowercase "don't"

Comment (yungez) [reply src/cloudShellRunner.ts]: I'll update it "Don't" given it's &

**Comment by yungez** [comment]
Issue #64

**Comment by yungez** [comment]
Issue #61  #66

**Comment by yungez** [comment]
#66 #64 #61

**Pull request merged**